source/dofs/dof_handler.cc

```diff
@@ -1766,15 +1770,17 @@ namespace internal
               // to have functions that can pack and unpack the data we want to
               // transport -- namely, the single unsigned int active_fe_index
               // objects
-              auto pack = [](const typename dealii::DoFHandler<dim, spacedim>::
-                               active_cell_iterator &cell) -> unsigned int {
+              auto pack =
+                [](const typename dealii::DoFHandler<dim, spacedim>::
+                     active_cell_iterator &cell) -> active_fe_index_type {
```

Comment:
```suggestion
                     active_cell_iterator &cell) {
```

Reply:
I prefer to be explicit. That also applies to return types :)